spec/requests/admin_checks_spec.rb

```diff
@@ -34,6 +36,46 @@
             expect(response.body).to include(claim.eligibility.current_school.name)
           end
         end
+
+        describe "checks#create" do
+          it "creates a new check and redirects to the next check" do
+            expect {
+              post admin_claim_checks_path(claim, check: "qualifications")
+            }.to change { Check.count }.by(1)
+
+            expect(claim.checks.last.name).to eql("qualifications")
+            expect(claim.checks.last.created_by).to eql(user)
+            expect(response).to redirect_to(admin_claim_check_path(claim, check: "employment"))
+          end
+
+          context "when the last check is marked as completed" do
+            let(:last_check) { Admin::ChecksController::CHECKS_SEQUENCE.last }
+
+            it "creates the check and redirects to the claims#show page" do
+              expect {
+                post admin_claim_checks_path(claim, check: last_check)
+              }.to change { Check.count }.by(1)
+
+              expect(claim.checks.last.name).to eql(last_check)
+              expect(claim.checks.last.created_by).to eql(user)
+              expect(response).to redirect_to(admin_claim_path(claim))
+            end
+          end
+
+          context "when a check has already been marked as completed" do
+            it "doesn't create a check and redirects with an error" do
+              create(:check, name: "qualifications", claim: claim)
+
+              expect {
+                post admin_claim_checks_path(claim, check: "qualifications")
+              }.not_to change { Check.count }
+
+              expect(response).to redirect_to(admin_claim_check_path(claim, check: "qualifications"))
+              follow_redirect!
```

Comment:
Not a major issue but cheaper to check the content of `flash[:alert]` instead of following the redirect and processing a second request.